tests/cupy_tests/linalg_tests/test_solve.py

```diff
@@ -33,6 +34,13 @@ def test_solve(self):
         self.check_x((2, 5, 5), (2, 5, 2))
         self.check_x((2, 3, 2, 2), (2, 3, 2,))
         self.check_x((2, 3, 3, 3), (2, 3, 3, 2))
+        old_limit = get_batched_gesv_limit()
+        set_batched_gesv_limit(0)  # disable use of batched_gesv
```

Comment:
Let's do this in setup/teardown better

Reply:
Updated to do that in setUp() and tearDown().